### ISSUE TITLE
Javascript Version - Web Based Cockatrice

### ISSUE THREAD

**Issue by LeoFC97** [Cockatrice/Cockatrice#3838]
What would be the difficulties of making a web version with javascript / node. My idea is to make a web version that connects to the same servers as the desktop version.

Any JS devs want to jump in this project with me?

**Comment by ctrlaltca** [comment]
A plain html and js concept already exists: https://github.com/Cockatrice/Cockatrice/tree/master/webclient

**Comment by LeoFC97** [comment]
Yes, but its 3+ years old, and nowdays the concept of js and node is another, I was thinking about a new repository totally independent on this.

**Comment by ZeldaZach** [comment]
I'd be down to see something like this. Please, feel free to reach out to us via [gitter](https://gitter.im/Cockatrice/Cockatrice) to chat. While we'd love to see a web client for Cockatrice, we just don't have the active development manpower to do so.

**Comment by ZeldaZach** [comment]
@seavor has spent some time working on this, feel free to reach out to him/us on [Discord](https://cockatrice.us/discord) for more information on how to build this further.

**Comment by tooomm** [comment]
There are now a dedicated set of repos for this, and the code from the initial first version moved from the main Cockatrice repo into their own sub-repos:

https://github.com/orgs/Cockatrice/repositories?q=lang%3Atypescript&type=all
